bigtable-hbase-parent/bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/adapters/read/TestScanAdapter.java

```diff
@@ -78,4 +103,31 @@ public void maxVersionsIsSet() {
             .build(),
         rowRequestBuilder.getFilter().getChain());
   }
+
+  @Test
+  public void testExtendedScan(){
+    byte[] row1 = Bytes.toBytes("row1");
+    byte[] row2 = Bytes.toBytes("row2");
+
```

Comment:
Would be good to test the boundary cases for binary keys: trailing 0xff and negatives